app.json

```diff
@@ -8,6 +8,14 @@
     "GOOGLE_CREDENTIAL": {
       "description": "Google Application Credential",
       "value": "xxxxxxx.apps.googleusercontent.com"
+    },
+    "ROOMS_DATA": {
+      "description": "List of avaliable rooms office. Read more in SETUP.md",
+      "value": ""
+    },
+    "ENFORCE_SSL": {
+      "description": "If you running in HTTPS this variable forces redirect to HTTPS when user access with HTTP",
+      "value": "true"
```

Comment:
não faria mais sentido o default ser false, pra facilitar subir o servidor sem `https`? o heroku, por exemplo, sobe um server com https?

Reply:
@jorgemdnt this file is to deploy heroku apps, that have ssl by default.
In my concern, SSL it's a best practice that should be encouraged.

Reply:
I'm really sorry, @juliemar & @alesshh . If that is the case, I got it wrong!